docs/hub/mlx.md

```diff
@@ -0,0 +1,100 @@
+# Using MLX at Hugging Face
+
+[MLX]([https://github.com/MaartenGr/BERTopic](https://github.com/ml-explore/mlx)) is a model training and serving framework on Apple Silicon made by Apple Research.
+
+It comes with variety of examples:
+
+- [Transformer language model](https://github.com/ml-explore/mlx-examples/tree/main/transformer_lm) training.
+- Large-scale text generation with [LLaMA](https://github.com/ml-explore/mlx-examples/tree/main/llms/llama) and
+- finetuning with [LoRA](https://github.com/ml-explore/mlx-examples/tree/main/lora).
+- Generating images with [Stable Diffusion](https://github.com/ml-explore/mlx-examples/tree/main/stable_diffusion).
+- Speech recognition with [OpenAI's Whisper](https://github.com/ml-explore/mlx-examples/tree/main/whisper).
+
+
+## Exploring MLX on the Hub
+
+You can find MLX models by filtering at the left of the [models page](https://huggingface.co/models?library=mlx&sort=trending).
+There's also [MLX community](https://huggingface.co/mlx-community) converting and publishing weights for MLX format.
+
+Thanks to MLX Hugging Face Hub integration, you can load MLX models with a few lines of code. 
```

Comment:
Maybe link to `Using Existing Models` here?

Reply:
Added at the end